docs/docs/behaviors/hold-tap.md

```diff
@@ -31,6 +31,12 @@ When the hold-tap key is released and the hold behavior has not been triggered,
 
 ![Hold-tap comparison](../assets/hold-tap/comparison.png)
 
+The 'hold-preferred' flavor tends to work well in combination with the shift, ctrl, caps lock, escape, tab and similar keys because they do not typically occur in the middle words or other rapid typing sequences in which the user may (unknowningly) execute a rolling keystrike, where they press the next key before the fully releasing the prior (hold/mod/layer-tap) key. Futhermore, because the decision to emit the hold behavior is made upon the second key being pressed, the 'hold-preferred' flavor most closely mimics the behavior of the shift key on a normal keyboard, and therefore tends to feel quite natural to users. However, because it biases towards the hold behavior in rolling keystrike situations, this flavor tends not to work well in combination with the alphas or for the spacebar, except for the small percentage of users that have very strict key-up habits.
+
+The 'tap-preferred' and 'balanced' flavors tend to work better in combination with the alphas or spacebar because they will emit the tap behavior if the key is involved in a rolling keystrike, as long as it happens quickly. This can work well with home-row modifiers.
```

Comment:
```suggestion
The 'tap-preferred' and 'balanced' flavors tend to work better in combination with the alphas or spacebar because they will emit the tap behavior if the key is involved in a rolling keystroke, as long as it happens quickly. This can work well with home-row modifiers.
```
See above.